SingletonExtensionFactoryTest will fail under windows

When building pf4j project under windows, org.pf4j.SingletonExtensionFactoryTest will fail due to path separator problem.
```
[INFO] Running org.pf4j.SingletonExtensionFactoryTest
2021-03-29 11:12:23,152 DEBUG org.pf4j.DefaultExtensionFactory - Create instance for extension 'org.pf4j.test.TestExtension'
2021-03-29 11:12:23,163 DEBUG org.pf4j.DefaultExtensionFactory - Create instance for extension 'org.pf4j.test.TestExtension'
2021-03-29 11:12:23,163 DEBUG org.pf4j.DefaultExtensionFactory - Create instance for extension 'org.pf4j.test.TestExtension'
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.016 s <<< FAILURE! - in org.pf4j.SingletonExtensionFactoryTest
[ERROR] createNewEachTimeFromDifferentClassLoaders  Time elapsed: 0.001 s  <<< ERROR!
java.lang.ClassNotFoundException: org.pf4j.test.TestExtension
        at org.pf4j.SingletonExtensionFactoryTest.createNewEachTimeFromDifferentClassLoaders(SingletonExtensionFactoryTest.java:62)
```

--- a/pf4j/src/test/java/org/pf4j/SingletonExtensionFactoryTest.java
+++ b/pf4j/src/test/java/org/pf4j/SingletonExtensionFactoryTest.java
@@ -74,7 +74,7 @@ public class SingletonExtensionFactoryTest {
     private URL[] getClasspathReferences() throws MalformedURLException {
         String classpathProperty = System.getProperty("java.class.path");
 
-        String[] classpaths = classpathProperty.split(":");
+        String[] classpaths = classpathProperty.split(File.pathSeparator);
         URL[] uris = new URL[classpaths.length];
 
         for (int index = 0; index < classpaths.length; index++) {

